CRM/Contact/Form/Relationship.php

```diff
@@ -219,10 +219,10 @@ public function setDefaultValues() {
       if (!empty($this->_values)) {
         $defaults['relationship_type_id'] = $this->_rtypeId;
         if (!empty($this->_values['start_date'])) {
-          list($defaults['start_date']) = CRM_Utils_Date::setDateDefaults($this->_values['start_date']);
+          $defaults['start_date'] = $this->_values['start_date'];
         }
         if (!empty($this->_values['end_date'])) {
-          list($defaults['end_date']) = CRM_Utils_Date::setDateDefaults($this->_values['end_date']);
+          $defaults['end_date'] = $this->_values['end_date'];
         }
```

Comment:
I think these statements are now redundant; you are basically assigning the same value that is already there.

Reply:
I've just checked and they are still required, _values is not passed to $defaults without assignment here.  However, I've cleaned it up so we're now using `CRM_Utils_Array::value()` instead of a 3 line if statement.